index.html

```diff
@@ -45,6 +45,24 @@
     </div>
 </div>
 
+<div class="csh-hacks-wrapper">
+    <div class="container spaced" id="homepage-about">
```

Comment:
You copied the id `homepage-about` from the about section, remove it. I'd just replace it with `csh-hacks-wrapper`, and target that id instead of a class, since there should only be one instance of this on a page.